stdlib/posix.pyi

```diff
@@ -99,21 +102,24 @@ P_ALL: int
 P_PGID: int
 P_PID: int
 
-RTLD_DEEPBIND: int
+if sys.platform != "darwin":
+    RTLD_DEEPBIND: int
 RTLD_GLOBAL: int
 RTLD_LAZY: int
 RTLD_LOCAL: int
 RTLD_NODELETE: int
 RTLD_NOLOAD: int
 RTLD_NOW: int
 
-SCHED_BATCH: int
 SCHED_FIFO: int
-SCHED_IDLE: int
 SCHED_OTHER: int
-SCHED_RESET_ON_FORK: int
 SCHED_RR: int
 
+if sys.platform != "darwin":
+    SCHED_BATCH: int
+    SCHED_IDLE: int
+    SCHED_RESET_ON_FORK: int
```

Comment:
`sched_setscheduler(2)` says "The SCHED_BATCH and SCHED_IDLE policies are Linux-specific."